src/components/create-guild/Requirements/components/ExistingRequirementEditableCard.tsx

```diff
@@ -98,7 +100,7 @@ const ExistingRequirementEditableCard = ({
         />
       </RequirementBaseCard>
 
-      {!isEditDisabled && (
+      {!isEditDisabled && !!FormComponent && (
```

Comment:
I think now we can remove the `!!FormComponent &&` check from `RequirementEditModal` ;)